scripts/docker/rhel/Dockerfile

```diff
@@ -0,0 +1,27 @@
+#
+# Copyright (c) .NET Foundation and contributors. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for full license information.
+#
+
+# Dockerfile that creates a container suitable to build dotnet-cli
+FROM microsoft/dotnet-buildtools-prereqs:rhel-7-rpmpkg-c982313-20174116044113
+
+# Install from sudo main package TODO This package needs to be mirrored
+RUN yum install -y https://www.sudo.ws/sudo/dist/packages/RHEL/7/sudo-1.8.20-3.el7.x86_64.rpm \
```

Comment:
@wli3 Assuming you are building this container on a RHEL 7 box registered with a subscription and receiving updates, the RHEL container should automatically be able to find yum from the right/default repositories. Can you find `sudo` using yum on the RHEL host?